Profiles/ProfilesAPI.py

```diff
@@ -1,6 +1,70 @@
-from flask import Blueprint
+from flask import Blueprint, make_response, request, Flask
 import sys
 from db import db
+from flask_cors import cross_origin
 sys.path.append("../")
 
-profiles_api = Blueprint("profiles", __name__)
\ No newline at end of file
+profiles_api = Blueprint("profiles", __name__)
+
+@profiles_api.route("/<int:id>", methods = ["GET"])
+@cross_origin(supports_credentials=True)
+def getProfile(id):
+    try:
+        name = db[id]["name"]
+        score = db[id]["scores"]
+        data = {"name": name, "scores": score}
```

Comment:
just return `data = db[id]` since qn2 asks you to POST name only. the person wont have `['scores']` in db, so the code will break.